bin/agat_sp_manage_functional_annotation.pl

```diff
@@ -1109,9 +1245,14 @@ =head1 OPTIONS
 Integer - Usefull only if -id is used.
 This option is used to define the number that will be used to begin the numbering. By default begin by 1.
 
+=item B<-a> or B<--addgntag>
+
+Add information in ouptut gff about if gene-name tag ('GN=') is present in blast db fasta ('gn_present=yes')
+or not ('gn_present=no'). Blast hits without entry in blast db will receive 'gn_present=no'.
```

Comment:
Blast hits without entry in blast db will receive 'gn_present=na'.
na instead of no if I well understood, isn't it?

Reply:
You are correct! Good catch. Corresct text should read
```
Blast hits without entry in blast db will receive 'gn_present=NA'.
```
`NA` since we don't know if the original fasta header had a GN or not! But we do (`yes` or `no`) if we have access to the blast db entry.